facdb/sql/hra_medicaid.sql

```diff
@@ -0,0 +1,31 @@
+DROP TABLE IF EXISTS _hra_medicaid;
+
+SELECT
```

Comment:
In our call I remember you mentioning you not knowing what's going on what medicaid.  This looks good to me.  What was your question or issue with medicaid?